packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/templates/disabledDynamicNavigation.html

```diff
@@ -6,18 +6,29 @@
   <template lwc:else>
     <template if:false={hideSort}>
       <div class="slds-var-m-bottom_small slds-clearfix">
-        <c-quantic-sort
-          class="slds-float_left"
-          engine-id={engineId}
-        ></c-quantic-sort>
+        <template lwc:if={shouldDisplaySort}>
```

Comment:
This was not updated accordingly.
```suggestion
        <template lwc:if={sortOptionsInitialized}>
```

Reply:
If this didn't make the E2E test fail, means that they can be improved x)
are you also testing the sort when the dne is disabled with the refine modal?